src/game/common/rts/specialpower.h

```diff
@@ -0,0 +1,63 @@
+/**
+ * @file
+ *
+ * @author Jonathan Wilson
+ *
+ * @brief Special Power
+ *
+ * @copyright Thyme is free software: you can redistribute it and/or
+ *            modify it under the terms of the GNU General Public License
+ *            as published by the Free Software Foundation, either version
+ *            2 of the License, or (at your option) any later version.
+ *            A full copy of the GNU General Public License can be found in
+ *            LICENSE
+ */
+#pragma once
+#include "always.h"
+#include "audioeventrts.h"
+#include "subsysteminterface.h"
+
+class SpecialPowerTemplate : public Overridable
+{
+    IMPLEMENT_POOL(SpecialPowerTemplate)
+
+public:
+    virtual ~SpecialPowerTemplate() override;
```

Comment:
Constructor is missing. Is that meant to be added in follow up change?

Reply:
None of the functions of this class need implementing yet so this just defines the virtual functions and the member variables.